### ISSUE TITLE
HTML sections

### ISSUE THREAD

**Issue by rimmesbe** [snakething/prj-rev-bwfs-dasmoto#1]
Notice how you have naturally split the HTML up into sections. Make it official and wrap those elements in divs:
<img width="122" alt="screen shot 2017-06-28 at 1 57 34 pm" src="https://user-images.githubusercontent.com/9957628/43208531-a931bd90-8ff8-11e8-8e35-c560b8ad60f7.png">
This makes it a lot easier to organize your code and style larger chunks of HTML.

https://github.com/snakething/prj-rev-bwfs-dasmoto/blob/master/Dasmoto/index.html#L12-L15

**Comment by snakething** [comment]
Yes i did it the same way in the start but i didn't see the reason to use it after i did the css to i took them a out i didnt add to html header and footer div as well
